src/components/navbar/navbar.jsx

```diff
@@ -2,9 +2,11 @@ import React from "react";
 import { Link } from "react-router-dom";
 import "./navbar.css";
 import { useCart } from "../../context/cartContext";
+import { useWish } from "../../context/wishContext";
```

Comment:
you can put multiple context file into single index file & import it from there

Reply:
ok